docs/JSON-RPC.md

```diff
@@ -280,7 +280,7 @@ Results:
 | result.city | string | The server city. |
 | result.countryId | number | The server country ID (see QLocale::Country). |
 | result.welcomeMessage | string | The server welcome message. |
-| result.directoryServer | string | The directory server to which this server requested registration, or blank if none. |
+| result.directoryServer | string | The directory to which this server requested registration, or blank if none. |
```

Comment:
```suggestion
| result.directoryServer | string | The directory where this server requested registration, or blank if none. |
```

I don't know if this is better. _To which_ is just a bit... _witchy_.

Reply:
I want to imply a request being sent _from_ the server _to_ the directory.  So the "to" helps, I think.  That then limits the rest of the phrase...

Reply:
Albatross, English! _where this server requested registration_ has no trouble indicating the actor.